packages/turbo-gen/src/generators/custom.ts

```diff
@@ -32,9 +34,25 @@ export async function generate({
         throw err;
       }
 
+      // make it obvious that we're done creating a generator, and now we're running it
+      console.log();
+      logger.info(`Example generator config successfully created!`);
+      logger.info(`Loading generator config...`);
+      console.log();
+
       // fetch generators again, and continue to selection prompt
       generators = getCustomGenerators({ project, configPath: opts.config });
+
+      // something went wrong and we weren't able to find our new demo generator
+      if (!generators.length) {
+        logger.error(`Error loading generator.`);
```

Comment:
Go linter tells us not to use `.` at the end of error messages, so we don't do that anywhere in the turbo CLI. Just calling out, not a big deal

Reply:
Good call